source/content/modules.md

```diff
@@ -34,9 +34,9 @@ This module provides general methods your site needs to access aspects of the in
 The code for the API module is available within our upstream repositories, in the `modules/pantheon` directory. The functionality of this module is provided in three parts: Apache Solr, Pantheon Platform API, and Login.
 
 ### [Pantheon Platform API](https://github.com/pantheon-systems/drops-7/blob/master/modules/pantheon/pantheon_api/pantheon_api.info)
-This module provides general methods your site needs to access aspects of the internal Pantheon API. This is necessary for clearing caches and other common workflows on the platform.
+This module provides general methods your site needs to access aspects of the internal Pantheon API. This is necessary for clearing caches and other common workflows on the platform. Currently, there is no API module necessary for Drupal 8.
```

Comment:
@greg-1-anderson should the previous sentence to your addition be modified to specify D7? As is, the two seem slightly contradictory.

Reply:
Ah, yeah. The path says "drops-7", but that is too subtle. This module is for Drupal 7 only.